tests/postgres_tests/test_integration.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import sys
+import unittest
+
+from django.test import SimpleTestCase
+
+try:
+    import psycopg2
+except ImportError:
+    psycopg2 = None
+
+
+@unittest.skipIf(psycopg2 is None, 'psycopg2 is required')
+class PostgresIntegrationTest(SimpleTestCase):
+    def test_management_command(self):
+        result = subprocess.run(
+            [sys.executable, '-m', 'django', 'check', '--settings', 'postgres_tests.integration_settings'],
```

Comment:
This fails when running from the root directory rather than the tests directory (i.e. `$ ./tests/runtests.py postgres_tests`). Also, the output doesn't make debugging very easy (`AssertionError: 1 != 0`) -- if that could be improved that could be nice.